src/components/app.tsx

```diff
@@ -42,20 +43,17 @@ export class AppComponent extends BaseComponent<IProps, IState> {
   }
 
   public render() {
-    const widthRatio = this.state.width / kAppWidth;
-    const heightRatio = this.state.height / kAppHeight;
-    let scalePercent = widthRatio > heightRatio ? heightRatio : widthRatio;
-    if (scalePercent > 1) {
-      scalePercent = 1;
-    }
```

Comment:
I assume that we want to scale over 100%.  No one ever commented on it, but this seems like the proper path.